### PR TITLE
update profile

### PR DESCRIPTION
**Pull request by hangxuka**

### CODE REVIEW

**Comment by VuDinhHao** [comment]
ok

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -27,7 +27,7 @@
               <%= t "header.account" %><b class="caret"></b>
             </a>
             <ul class="dropdown-menu">
-              <li><%= link_to t("static_pages.header.profile"), "#" %></li>
+              <li><%= link_to t("static_pages.header.profile"), edit_user_path(current_user) %></li>
               <li><%= link_to t("static_pages.header.setting"), edit_user_registration_path(current_user) %></li>
```
Comment (TaoLuyen) [inline app/views/layouts/_header.html.erb]: hon 80 ki tu :3

File: config/routes.rb [hunk 2]
```diff
@@ -16,6 +16,6 @@
     resources :users
   end
   resources :books
-  resources :users, only: [:index]
+  resources :users
```
Comment (TaoLuyen) [inline config/routes.rb]: dung cai nao only cai do

**Comment by TaoLuyen** [comment]
ok

**Pull request merged**